src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs

```diff
@@ -177,7 +177,7 @@ public void reader_14()
             ReadFrom(xml);
         }
 
-        //XElement: non-wellformed xml
+        //XElement: non�well-formed xml
```

Comment:
Ah, it didn't get eaten, but it got saved encoded as something that this view isn't showing.